src/components/atoms/interactiveAtom.tsx

```diff
@@ -23,6 +24,18 @@ const InteractiveAtomStyles = (pillarStyles: PillarStyles): SerializedStyles =>
         border-bottom: 0.0625rem solid ${neutral[86]};
     }
 `;
+const atomCss = `
+    ${pageFonts} 
+    body {
+        background: white !important;
+        padding: ${remSpace[2]} !important;
```

Comment:
These styles should be wrapped in a `${darkModeCss``}`

Reply:
Actually these styles are just a string and aren't wrapped in ``` css`` ``` so you would need:

`@media (prefers-color-scheme: dark) {`